client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -72,26 +97,20 @@ export default function IssueMultipleLicensesForm( {
 
 	const onSelectProduct = useCallback(
 		( product ) => {
+			// A bundle cannot be combined with other products.
+			if ( isJetpackBundle( product.slug ) ) {
+				return;
```

Comment:
I think we will still have to set the bundle for assigning the bundle to work.

Reply:
Good catch, I have fixed this.